src/view/pages/Influencers/PersonalInformation/Job.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import Button from "../../../../components/Button/Button";
+import classNames from "classnames/bind";
+import styles from "./Job.module.scss";
+
+const cx = classNames.bind(styles);

```

Comment:
What does `cx` mean? Should be named clearly for others to easily understand!

Reply:
For easier reading when we use multiple variables for className.
Ex: <button className={`btn btn__summit ${isDangerField ? 'btn--danger' : ''} ${isRoundedField ? 'btn--rounder' : ''} ${isDisabled ? 'btn-disable' : ''}`}>Submit</button>
=> very messy
<button className={classnames(
       'btn',
       'btn__summit',
       { 'btn--danger': isDangerField },
       { 'btn--rounder': isRoundedField },
       { 'btn-disable': isDisabled }
     )}>Submit</button>
=> more clear